### PR TITLE
Ma/fix logs

### PR DESCRIPTION
**Pull request by markusahlstrand**
Small patch to be more explicit with the logs on the user. Before it just said "Updated profile", now it says: "Login with apple". Updated wrangler to get rid of warning messages.

### CODE REVIEW

File: scripts/upload-r2-prod.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export NO_D1_WARNING=true
 cd src
 find templates -name "*.*" | while read fname; do
-    wrangler r2 object put auth-templates/$fname -f $fname --env production
+    npx wrangler r2 object put auth-templates/$fname -f $fname --env production
```
Comment (markusahlstrand) [inline scripts/upload-r2-prod.sh:4]: Added npx here so it's the same is in dev. This actually doesn't work which is annoying. Get a 500 from cloudflare. Had to drop the file in R2 by hand.

File: src/models/User.ts [hunk 2]
```diff
@@ -240,6 +240,69 @@ export const userRouter = router({
...
+      });
+
+      await writeLog(ctx.state.storage, {
+        category: "login",
+        message: `Login with ${input.connection.name}`,
+      });
+
+      return profile;
+    }),
+  patchProfile: publicProcedure
```
Comment (markusahlstrand) [inline src/models/User.ts:270]: This is just moved up a bit in the file so they are alphabetically ordered..

File: src/routes/tsoa/passwordless.ts [hunk 3]
```diff
@@ -50,10 +50,6 @@ export class PasswordlessController extends Controller {
       },
     });
 
-    console.log("created code ok");
```
Comment (markusahlstrand) [inline src/routes/tsoa/passwordless.ts:53]: Found some old debug messages.

File: src/.DS_Store
Comment (danielgent) [inline src/.DS_Store:1]: What is this?

**Review by danielgent — APPROVED** [review]

File: src/.DS_Store
Comment (markusahlstrand) [reply src/.DS_Store:1]: This is a nasty file the should be in the .gitignore.. Some temporary file on macs.

**Pull request merged**